recon_surf/recon-surf.sh

```diff
@@ -540,18 +518,31 @@ echo " " |& tee -a $LF
 echo "================== Creating orig and rawavg from input =========================" |& tee -a $LF
 echo " " |& tee -a $LF
 
+CONFORM_LF=$SUBJECTS_DIR/$subject/scripts/conform.log
+if [ $CONFORM_LF != /dev/null ] ; then  rm -f $CONFORM_LF ; fi
+echo "Log file for Conform test" > $CONFORM_LF
+
 # check for input conformance
-cmin=""
-if  [ "$vox_size" == "auto" ]
+cmd="$python ${binpath}../FastSurferCNN/data_loader/conform.py -i $t1 --check_only --vox_size min --verbose"
```

Comment:
Why is min passed here, but in the call below we pass the exact voxelsize? To check for conformity between the two?

Reply:
1. It checks conformity between the two.
2. We don't need to do a bunch of error checking if we got the right output in the first call (i.e., if the output of call #1 or the cut-commands were unclear, the second call will fail)